packages/flutter_tools/lib/src/build_system/targets/web.dart

```diff
@@ -413,8 +413,8 @@ const CORE = [
 self.addEventListener("install", (event) => {
   return event.waitUntil(
     caches.open(TEMP).then((cache) => {
-      // Provide a 'reload' param to ensure the latest version is downloaded.
-      return cache.addAll(CORE.map((value) => new Request(value, {'cache': 'reload'})));
+      return cache.addAll(
+        CORE.map((value) => new Request(value + '?v=' + RESOURCES[value], {'cache': 'reload'})));
```

Comment:
Will it be more helpful for the developer if we used a descriptive URL parameter name? E.g. "revision", "version", "cache_key"?

Reply:
good idea, will use `revision`